subsys/mgmt/Kconfig

```diff
@@ -59,6 +59,7 @@ config MCUMGR_SMP_UART_MTU
 
 endif
 
+if MCUMGR
```

Comment:
aren't all options in this file depending on MCUMGR?

Reply:
No, not those being ifdefed at least.